lib/qa/authorities/authority_with_sub_authority.rb

```diff
@@ -1,12 +1,18 @@
 module Qa::Authorities
-  class AuthorityWithSubAuthority < Base
```

Comment:
This is a pretty big breaking change isn't it? I don't think we have any classes that were dependent on this in OD, but I'm not sure what external implementation is like. Is it possible to deprecate?